src/style.css

```diff
@@ -1 +1,66 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+body {
+  margin: 0;
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+}
+
+.navigation {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 1200px;
```

Comment:
if you hardcoded the width of the header you also should center it by `margin: 0 auto;`